src/routing/command.rs

```diff
@@ -59,6 +59,7 @@ pub(crate) enum Command {
         recipients: Vec<SocketAddr>,
         delivery_group_size: usize,
         message: Bytes,
+        to_client: bool,
```

Comment:
While this works, I think it would be a bit cleaner to have a separate command for that. Because when sending to a client, we don't use `delivery_group_size` and `recipients` has always exactly one element so this makes it ever so slightly prone to bugs. So say:

```rust
enum Command {
    // ...
    SendMessageToNodes {
        recipients: Vec<SocketAddr>,
        delivery_group_size: usize,
        message: Bytes,
    },
    SendMessageToClient {
        recipient: SocketAddr,
        message: Bytes,
    }
}
```
Up to you though.

Reply:
right, I will make them two separate commands